.editorconfig

```diff
@@ -1,10 +0,0 @@
-# EditorConfig: automatically set indentation and such
-# http://EditorConfig.org
-
-# top-most EditorConfig file
-root = true
-
-[*.{js,json}]
-charset = utf-8
-indent_style = space
-indent_size = 2
```

Comment:
I prefer keeping the style configuration in one place (prettier's config), and `husky` and `lint-staged` do a good job at keeping everything formatted before pushing changes.

I could still restore this file, since most editors respect `.editorconfig`s without any configuration.